lib/foreman_discovery/version.rb

```diff
@@ -1,3 +1,3 @@
 module ForemanDiscovery
-  VERSION = "21.0.1"
+  VERSION = "21.0.2"
```

Comment:
Shouldn't we bump the version to `22.0.1` since we bumping also the required version of Foreman?

Reply:
Looks like with version 20 it was bumped to `requires_foreman '>= 3.2'`: https://github.com/theforeman/foreman_discovery/commit/fac49e346c17fc0853f651fb5b4bccfdbd96eb97

so this bump in `requires_foreman` should have happened when we moved to discovery v21

Reply:
Ok, but still if we bumping it now, we need to move to `22`. Changing `0.0.X` version means we are changing some small stuff that is backwards compatible

Reply:
I guess you are right @stejskalleos, any other thoughts @ares?

Reply:
Failing tests are fixed here: https://github.com/theforeman/foreman/pull/9288
If you update version to `22` we will have :green_circle: run and can merge